embc-app/ClientApp/src/app/volunteer/pages/referral-maker/referral-maker.component.ts

```diff
@@ -182,6 +184,8 @@ export class ReferralMakerComponent implements OnInit {
   addFoodReferral() {
     const referral: FoodReferral = {
       id: null, // is populated by BE after save
+      referralId: null, // is populated by BE after save: The actual property returned
+      essNumber: null,
       active: true,
       type: 'FOOD',
       subType: null,
```

Comment:
This is separate from this PR, but you could remove this line setting the subType (since it's an optional interface property).